react_native/src/pages/my-page/VerifyPage.tsx

```diff
@@ -0,0 +1,95 @@
+import { semanticNumber } from '@/styles/semantic-number';
+import { semanticFont } from '@/styles/semantic-font';
+
+import IconChevronLeft from '@/assets/icons/IconChevronLeft.svg';
+import IconX from '@/assets/icons/IconX.svg';
+import { semanticColor } from '@/styles/semantic-color';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { RootStackParamList } from '@/types/navigation';
+
+function VerifyPage() {
+  const isDone = false;
+  const navigation = useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+
+  return (
+    <View style={styles.verifyPageContainer}>
+      <CenterHeader
+        title={!isDone ? '개인 정보 수정' : ''}
+        leftChilds={
+          !isDone
+            ? {
+                icon: <IconChevronLeft width={28} height={28} stroke={semanticColor.icon.primary} strokeWidth={2} />,
+                onPress: () => navigation.goBack(),
+              }
+            : undefined
+        }
+        rightChilds={
+          isDone
+            ? [
+                {
+                  icon: <IconX width={28} height={28} stroke={semanticColor.icon.primary} strokeWidth={2} />,
+                  onPress: () => navigation.navigate('VerifyInfoPage'),
+                },
+              ]
+            : undefined
+        }
+      />
+      <View style={styles.verifyPageContainer}>
+        <View style={styles.textWrapper}>
+          <Text style={styles.mainText}>{!isDone ? '개인 정보를 수정하시겠어요?' : '본인 인증이 완료되었어요.'}</Text>
+          <Text style={styles.subText}>
+            {!isDone ? '개인 정보를 수정하려면 본인 인증을 진행해야 합니다.' : '개인 정보가 수정되었습니다.'}
+          </Text>
+        </View>
+        <View style={styles.chipWrapper}>
+          <Chip text={!isDone ? '본인 인증 필요' : '본인 인증 완료'} variant="condition" />
+        </View>
+      </View>
+      <View style={styles.buttonWrapper}>
+        <MainButton
+          children={!isDone ? '본인 인증하기' : '확인'}
+          onPress={!isDone ? () => navigation.navigate('VerifyInfoPage') : () => navigation.navigate('VerifyInfoPage')}
+        />
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  verifyPageContainer: {
+    flex: 1,
+    backgroundColor: semanticColor.surface.white,
+  },
+  textWrapper: {
+    paddingHorizontal: semanticNumber.spacing[16],
+    paddingTop: semanticNumber.spacing[40],
+    paddingBottom: semanticNumber.spacing[12],
+    gap: semanticNumber.spacing[4],
+  },
+  mainText: {
+    ...semanticFont.headline.medium,
+    color: semanticColor.text.primary,
+  },
+  subText: {
+    ...semanticFont.body.large,
+    color: semanticColor.text.tertiary,
+  },
+  chipWrapper: {
+    paddingHorizontal: semanticNumber.spacing[16],
+    paddingTop: semanticNumber.spacing[24],
+  },
+  buttonWrapper: {
+    paddingHorizontal: semanticNumber.spacing[16],
+    paddingTop: semanticNumber.spacing[10],
+    paddingBottom: semanticNumber.spacing[36],
```

Comment:
```suggestion
    paddingVertical: semanticNumber.spacing[10],
```
해당 부분에 QA 반영이 덜 된 것 같습니다!